app/[lang]/dictionaries.ts

```diff
@@ -0,0 +1,50 @@
+const dictionaries: LangDictionary = {
+  en: () => import('./dictionaries/en.json').then((module) => module.default),
+  fr: () => import('./dictionaries/fr.json').then((module) => module.default),
+  kw: () => import('./dictionaries/kw.json').then((module) => module.default),
+};
+
+export const getDictionary = async (lang: Lang) => {
+  const dictionaryFunction = dictionaries[lang];
+  if (typeof dictionaryFunction === 'function') {
```

Comment:
Is this for making TypeScript stop complaining? Instead of doing `return dictionaries[lang]()`?

Reply:
I tried `return dictionaries[lang]()` initially, but it did not work. Tested again now and somehow it works. Simplified it accordingly.